benchmarks/README.rst

```diff
@@ -69,6 +69,26 @@ Command-line help is available as usual via ``asv --help`` and
 
 .. _ASV documentation: https://asv.readthedocs.io/
 
+Benchmarking versions
+---------------------
+
+To benchmark or visualize only releases on different machines locally, the tags with their commits can be generated, before being run with ``asv``, that is::
+
+    cd benchmarks
+    # Get commits for tags
+    # delete tag_commits.txt before re-runs
+    for gtag in $(git tag --list --sort taggerdate | grep "^v"); do
+    git log $gtag --oneline -n1 --decorate=no | awk -v gtg="$gtag" '{print $1, gtg;}' >> tag_commits.txt
+    done
+    # Take last 20
+    for commitLine in $(tail --lines=20 tag_commits.txt); do
+    asv run $(echo $commitLine | awk '{print $1}')^!
+    done
+    # Publish and view
+    asv publish
+    asv preview
+
+Note that this can still be a significant time commitment. Do not open pull requests to the main repository with these results.
```

Comment:
I am not sure what the intent of this line is. In general benchmarking on NumPy is slow, this technique will run 20 very slow things. That should be clear to anyone running benchmarks. 

Why would someone open a PR with these results?

Reply:
I was thinking if we had a separate repo, since the results are static we could have people commit reasonably run benchmarks on different hardware. Though, yes that doesn't make sense here. I'll add it to the `asv publish` results repo.

Reply:
sounds good